rclpy/rclpy/node.py

```diff
@@ -295,5 +295,16 @@ def get_service_names_and_types(self):
     def get_node_names(self):
         return _rclpy.rclpy_get_node_names(self.handle)
 
+    def _count_publishers_or_subscribers(self, topic_name, func):
+        fq_topic_name = expand_topic_name(topic_name, self.get_name(), self.get_namespace())
+        validate_topic_name(fq_topic_name)
+        return func(self.handle, fq_topic_name)
+
+    def count_publishers(self, topic_name):
```

Comment:
Mind adding a doc string to this and `count_subscribers`? I realize most functions here don't have one, but adding them to new functions is an improvement.

Reply:
66bdf9c